client/src/pages/Explorer/utils.js

```diff
@@ -0,0 +1,28 @@
+export const shuffleFunction = (array) => {
+  let currentIndex = array.length;
+  let temporaryValue;
+  let randomIndex;
+
+  while (0 !== currentIndex) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    // And swap it with the current element.
+    temporaryValue = array[currentIndex];
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
+  }
+  return array;
+};
+
+export const getUpdatedList = async (cityList, favoriteList) => {
+  const filteredFav = cityList.filter((place) =>
+    favoriteList.includes(place.name),
+  );
+  const unLikesPlaceName = cityList.filter(
+    (place) => !favoriteList.includes(place.name),
+  );
+  const shuffledArray = await shuffleFunction(unLikesPlaceName);
```

Comment:
shuffleFunction is not async so this await shouldn't make a difference and getUpdatedList shouldn't need to be async